src/components/JobCard.js

```diff
@@ -5,7 +5,7 @@ import {Row, Col,Badge} from "react-bootstrap"
 import moment from 'moment'
 import {useHistory} from 'react-router-dom'
 
-export default function JobList(props) {
+export default function JobCard(props) {
     let item = props.job
```

Comment:
let's be consistent about `;`. If we use, we use. If we don't, we don't. 

https://github.com/airbnb/javascript#semicolons 

Both AirBnb and Google recommend using semicolons; let's just use them.

Reply:
here for naming, i don't like doing `let item = props.job;` because `item` is a bad variable name, but also it introduces one more confusing variable - we already have `props.job`, so it's clearer if we just keep that name throughout. The easy way to do this is to use the deconstructor: 
```
export default function JobCard({job}) { 
}
```